docs/pages/installation.md

```diff
@@ -1,59 +1,177 @@
 ---
 title: Installation
-description: There are many ways to install Foundation, but if you're just getting started, we have a few suggestions.
+description: There are many ways to install Foundation from downloading the compiled CSS, automated Sass installs to popular package managers. Each one serves a different skill level or preferred workflow. If this is all new to you, don’t worry; The video below will give you an overview of each method and help you decide which is best for you.
 video: '6KwsGcEHVTE'
 ---
 
-### Command-Line Tool
+## How to choose the right version for you?
+
+Depending on your skill level or how quick you want to get started, each of these different ways to get started might suit you better. Here’s a breakdown of the options you’ll see on this page:
+
+<ul class="no-bullet" data-magellan data-bar-offset="90">
+
+  <li><a href="#download-the-css-starter-template"><strong>Download the CSS Starter Template</strong></a> - A ready-to-go zip file with Foundation’s CSS and JS. <span style="vertical-align: top; border: 0; border-radius: 3px;" class="label secondary">CSS</span></li>
+
+  <li><a href="#cdn-links"><strong>Use the Foundation CDN</strong></a> - Foundation’s CDN (Content Delivery Network) can be linked into a new or existing project without any downloading. <span style="vertical-align: top; border: 0; border-radius: 3px;" class="label secondary">CSS</span></li>
+
+  <li><a href="#command-line-tool"><strong>Foundation CLI </strong></a> - Foundation’s CLI (Command Line Interface) is the recommended way to create ready-to-go Foundation Sass projects. <span style="vertical-align: top; border: 0; border-radius: 3px;" class="label">Sass</span></li>
+
+  <li><a href="#manual-setup"><strong>Manual Setup</strong></a> - You can Install from either the ZURB Stack or Basic Template manually. <span style="vertical-align: top; border: 0; border-radius: 3px;" class="label">Sass</span></li>
+
+  <li><a href="#package-managers"><strong>Package Managers</strong></a> - Install foundation through your favorite package manager like npm, Bower, Meteor, RubyGems, Composer or NuGet. <span style="vertical-align: top; border: 0; border-radius: 3px;" class="label">Sass</span></li>
+
+  <li><a href="#yeoman-generator"><strong>Yoeman Generators</strong></a> A generator is basically a plugin to scaffold complete projects or useful parts.
+ <span style="vertical-align: top; border: 0; border-radius: 3px;" class="label">Sass</span></li>
+
+  <li><a href="#download-the-sass-files"><strong>Download the Sass files</strong></a> - Already have a workflow you like? No problem, download Foundation's core SCSS files and you can go your own way. <span style="vertical-align: top; border: 0; border-radius: 3px;" class="label">Sass</span></li>
```

Comment:
added this section! I think we need to package up a Sass download with the essentials rather than the whole framework @kball